<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>669</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>762</width>
    <height>606</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>751</width>
      <height>601</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>описание проекта</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>741</width>
        <height>471</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>741</width>
         <height>431</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>621</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Здесь вы можете оставить описание вашего проекта,записать новые идеи для реализации </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Риски</string>
     </attribute>
     <widget class="QGroupBox" name="Box_Risk">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>741</width>
        <height>521</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QTableWidget" name="tableWidget_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>721</width>
         <height>431</height>
        </rect>
       </property>
       <property name="rowCount">
        <number>4</number>
       </property>
       <property name="columnCount">
        <number>4</number>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>176</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>51</number>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>69</number>
       </attribute>
       <row/>
       <row/>
       <row/>
       <row/>
       <column>
        <property name="text">
         <string>название риска</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>описание</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>вероятность</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>возможные потери</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>431</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Здесь необходимо указать все возможные риски при реализации вашего проекта</string>
       </property>
      </widget>
      <widget class="QPushButton" name="add_risk">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>490</y>
         <width>121</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>добавить риск</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Ресурсы</string>
     </attribute>
     <widget class="QGroupBox" name="Box_Resources">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>741</width>
        <height>521</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QTableWidget" name="tableRes">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>721</width>
         <height>391</height>
        </rect>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <property name="rowCount">
        <number>1</number>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>353</number>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>25</number>
       </attribute>
       <row/>
       <column>
        <property name="text">
         <string>название ресурса</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>описание ресурса</string>
        </property>
       </column>
      </widget>
      <widget class="QPushButton" name="add_res">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>470</y>
         <width>131</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>добавить ресурс</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>351</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>необходимо добавть ресурсы, требуемые для выполнения проекта</string>
       </property>
      </widget>
      <widget class="QPushButton" name="edit_res">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>470</y>
         <width>111</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>редактировать</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>План</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>741</width>
        <height>513</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QPushButton" name="add_task">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>480</y>
         <width>121</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Добавить задачу</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>351</width>
         <height>421</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="rowCount">
        <number>6</number>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>128</number>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>27</number>
       </attribute>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column>
        <property name="text">
         <string>Номер</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Задача</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Продолжительность</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Предыдущая задача</string>
        </property>
       </column>
      </widget>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>40</y>
         <width>351</width>
         <height>421</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>10</y>
         <width>451</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>В данном разделе вы можете добавлять задачи для дальнейшего планирования</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>доска KANBAN</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>751</width>
        <height>571</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QGraphicsView" name="graphicsView_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>741</width>
         <height>491</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>0</y>
         <width>601</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>  здесь вы сможете следить за тем, на каком этапе работы находятся ваши сотрудники</string>
       </property>
      </widget>
      <widget class="QPushButton" name="createKanban">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>530</y>
         <width>171</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>создать доску KANBAN</string>
       </property>
      </widget>
      <zorder>graphicsView_2</zorder>
      <zorder>label_4</zorder>
      <zorder>createKanban</zorder>
      <zorder>createKanban</zorder>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>762</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>справка</string>
    </property>
    <addaction name="action"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>файл</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menu_3"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>о программе</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Загрузить</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>выход</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
